examples/time_frequency/plot_epochs_spectra.py

```diff
@@ -43,3 +46,12 @@
 
 # Now let's take a look at the spatial distributions of the psd.
 epochs.plot_psd_topomap(ch_type='grad', normalize=True)
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+psds_mt, freqs_mt = psd_multitaper(epochs[:5], low_bias=True, tmin=tmin,
+                                   tmax=tmax, picks=[1], n_jobs=1)
+f, ax = plt.subplots()
+ax.plot(freqs_mt, 10 * np.log10(psds_mt[:, 0, :].T))
```

Comment:
that was why I added plt.ion before. I think it's because the methods for plotting automatically call plt_show under the hood. I can just add that in there.